x-pack/plugin/security/src/test/java/org/elasticsearch/xpack/security/remoteaccess/RemoteAccessAuthenticationServiceIntegTests.java

```diff
@@ -136,7 +136,7 @@ public void testInvalidHeaders() throws IOException {
         }
 
         try (var ignored = threadContext.stashContext()) {
-            Authentication authentication = AuthenticationTestHelper.builder().apiKey().build();
+            Authentication authentication = AuthenticationTestHelper.builder().serviceAccount().build();
```

Comment:
At the moment, service accounts are not supported as the remote access authentication subject. So I'm using service account to make this test pass. But, I think we should change this in the follow up.